plugin/path_config.go

```diff
@@ -114,10 +114,19 @@ func (b *backend) configUpdateOperation(ctx context.Context, req *logical.Reques
 	maxTTL := fieldData.Get("max_ttl").(int)
 	lastRotationTolerance := fieldData.Get("last_rotation_tolerance").(int)
 
-	length := fieldData.Get("length").(int)
-	formatter := fieldData.Get("formatter").(string)
 	passwordPolicy := fieldData.Get("password_policy").(string)
 
+	var length int
+	if lengthRaw, ok := fieldData.GetOk("length"); ok {
+		length = lengthRaw.(int)
+	} else if !ok && passwordPolicy == "" {
```

Comment:
nit: could drop `!ok` since `ok` always false here

Reply:
Removed in [53dc05d](https://github.com/hashicorp/vault-plugin-secrets-ad/pull/85/commits/53dc05df3258092d33853c9ca2a744e5b433d838), thanks!